ee/codegen/src/serializers/vellum.ts

```diff
@@ -585,10 +585,9 @@ export declare namespace BaseDisplayableWorkflowNodeSerializer {
 
 export const EntrypointNodeSerializer: ObjectSchema<
```

Comment:
I'd also argue we don't need these `ObjectSchema` annotations here at all, except for maybe in the case of recursion

Reply:
They're def needed for recursion – I haven't figured out at what level of the recursion they're minimally needed, so got into the habit of just added everywhere.